src/app/portlets/dot-edit-page/content/services/html/dot-edit-content-toolbar-html.service.ts

```diff
@@ -202,14 +225,39 @@ export class DotEditContentToolbarHtmlService {
                             uuid: container.dataset.dotUuid
                         },
                         disabled: isDisabledFormAdd,
-                        tooltip: isDisabledFormAdd
-                            ? this.dotMessageService.get('dot.common.license.enterprise.only.error')
-                            : ''
+                        tooltip: this.getTooltipErrorMessage(
+                            isDisabledFormAdd,
+                            maxContentletsLimitReached
+                        )
```

Comment:
I think all this code is very weird and not readable enoght, maybe it is better do something like:

```suggestion
                     let validationError: ValidationError = getContentletValidationError(item);
                     ...
                     tooltip: validationError === ValidationErrorMessage(validationError)
                     ...
                     getContentletValidationError(item): ValidationError {
                          if (item === 'form' && !this.isEnterpriseLicense) {
                              return ValidationError.EnterpriseLicenseError;
                          } else if (this.isMaxContentletsLimitReached(container)) {
                              return ValidationError.MaxContentletsLimitReacheError;
                          }
                     }
```

where ValidationError is a enum